Src/java/quick/src/main/resources/org/hl7/fhir/fhir-modelinfo-4.0.0.xml

```diff
@@ -3782,7 +3782,7 @@
             <ns4:elementTypeSpecifier xsi:type="ns4:ListTypeSpecifier" elementType="FHIR.Coding"/>
         </ns4:element>
     </ns4:typeInfo>
-    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="BodyStructure" retrievable="true" baseType="FHIR.DomainResource">
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="BodyStructure" retrievable="true" baseType="FHIR.DomainResource" primaryCodePath="morphology">
```

Comment:
I'd actually expect the `primaryCodePath` to be `location`. (FHIR 4.0.0 `BodyStructure`)

Reply:
Morphology seems more like the "kind" of thing, where "location" is more the thing, though I don't feel strongly, that's why I picked Morphology (the FHIR description says "kind of structure").

Reply:
This is probably a matter of preference.  Looking at the three examples provided for `BodyStructure`, we have this:

Example | Morphology | Location
------------ | -------------  | -------------
fetus | _(none)_ | SCT#83418008 Entire fetus (body structure)
tumor | SCT#4147007 Mass (morphologic abnormality) | SCT#78961009 Splenic structure (body structure)
skin-patch | _(text only, no code)_ Skin patch | SCT#14975008 forearm |

To me, when identifying a body structure, I'm more likely to think of it in terms of the locations in the examples rather than the morphologies in the examples.  But like I said, it's likely a matter of preference -- so if you disagree, I'm fine with leaving as-is.

Reply:
Okay, agreed, changed to location.